Assets/Scripts/Bundle.cs

```diff
@@ -0,0 +1,79 @@
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+namespace FinalInferno {
+    [System.Serializable]
+    public class Bundle<T> where T : ScriptableObject, IDatabaseItem {
+        // Lista serializavel configurada pelo editor
+        // TO DO: Adicionar HideInInspector depois que tiver certeza que funciona
+        [SerializeField] private List<T> assets = new List<T>();
+
+        private string bundleName = "";
+        public string BundleName => bundleName;
+
+        private Dictionary<string, T> dict = new Dictionary<string, T>();
+        private bool loaded = false;
+
+        public Bundle() {
+            bundleName = typeof(T).Name.ToLower();
+        }
+
+#if UNITY_EDITOR
+        public void InitializeAsset() {
+            string[] guidAssetsPath = LocateAssets();
```

Comment:
```suggestion
            string[] assetsGUIDs = LocateAssets();
```
theses strings are the GUIDs of each asset, not their paths